deploy/200-role.yaml

```diff
@@ -21,7 +21,11 @@ metadata:
 rules:
 - apiGroups: ['shipwright.io']
   resources: ['buildruns']
-  verbs:     ['get', 'list', 'update', 'watch']
+  verbs:     ['get', 'list', 'watch', 'update']
+
+- apiGroups: ['shipwright.io']
```

Comment:
Can you add comments about why these finalizer permissions are needed? Gabe's PR had some more description: https://github.com/shipwright-io/build/pull/808/files

Reply:
I'd rather phrase these upstream comments in terms of which admission controllers we enable on the KinD cluster we set up during CI, which may or may not correspond to specific vendor-specific platforms.

If there's more breaking movement in this area we might consider having 2+ KinD configs we run during CI, and ensure that this single config covers both, but I don't think we need to do that just yet.

Reply:
agreed ... 4.7 / 4.8 most likely is translated to k8s 1.20 vs. 1.21 or whatever is the 4.7 to 4.8 difference in k8s versions

Reply:
by the way, when attempting to run the shipwright integration tests on openshift 4.8, I discovered the need for additional permissions that did not arise when simply creating a buildrun/taskrun  .... so please consider me finally "up to date / on  the same page" :-) and disregard my call for version distinctions.

And just add a general "why this is needed comment" like @imjasonh asked for my my now closed, subset PR #808

Reply:
Comments added to file.